tests/test_androidndk.py

```diff
@@ -0,0 +1,136 @@
+import unittest
+import sys
+
+from pythonforandroid.androidndk import AndroidNDK
+
+
+class TestAndroidNDK(unittest.TestCase):
+    """
+    An inherited class of `unittest.TestCase`to test the module
+    :mod:`~pythonforandroid.androidndk`.
+    """
+
+    def setUp(self):
+        """Configure a :class:`~pythonforandroid.androidndk.AndroidNDK` so we can
+        perform our unittests"""
+        self.ndk = AndroidNDK("/opt/android/android-ndk")
+
+    def test_host_tag(self):
+        """Test the `host_tag` property of the :class:`~pythonforandroid.androidndk.AndroidNDK`
+        class."""
+        if sys.platform == "linux":
+            self.assertEqual(self.ndk.host_tag, "linux-x86_64")
+        elif sys.platform == "darwin":
+            self.assertEqual(self.ndk.host_tag, "darwin-x86_64")
```

Comment:
Why don't we `mock.patch()` the sys.platform ourselves just for the test?

Reply:
Why not? Addressed 👍